CHANGELOG.md

```diff
@@ -3,52 +3,99 @@
-  Implement `Elem` type family!
-* [#203](https://github.com/kowainik/relude/issues/203):
-  Implement `guarded` takes a predicate and a value and returns an Alternative.
-* [#201](https://github.com/kowainik/relude/issues/201):
-  Implement `!!?` as a safe equivalent of `!!` that returns a `Maybe`!
-* [#192](https://github.com/kowainik/relude/issues/192):
-  Reexport `foldMap'` from `Data.Foldable`.
-* [#187](https://github.com/kowainik/relude/issues/187):
-  Remove `tasty` and `tasty-hedgehog` dependencies and their redundant imports.
-* [#195](https://github.com/kowainik/relude/pull/195):
-  Implement `foldMap1` for `NonEmpty` in terms of `foldr`.
-* [#194](https://github.com/kowainik/relude/pull/194):
-  Use `mapToFst` instead of `zip` to improve list fusion in `inverseMap`.
-* [#193](https://github.com/kowainik/relude/pull/193):
-  Implement `andM`, `orM`, `allM`, and `anyM` in terms of `&&^` and `||^`.
-* [#191](https://github.com/kowainik/relude/pull/191):
-  Implement `asumMap` and `foldMapA` by coercing `foldMap`.
-  BREAKING CHANGE: Reorder type parameters to `asumMap`.
-* Use `$>` instead of `*>` and `pure` where possible.
-* [#167](https://github.com/kowainik/relude/issues/167):
-  Rename functions `prec`/`prev`, `dupe`/`dup`.
-* [#155](https://github.com/kowainik/relude/issues/155):
-  Implement `Relude.Extra.Foldable` module.
-* Re-export `GHC.Float.atan2`.
-* [#172](https://github.com/kowainik/relude/issues/172):
-  Add `Monoid` and `Semigroup` instances for `Validation` type
-* [#177](https://github.com/kowainik/relude/issues/177):
-  Improve usage of performance pragmas.
-* [#178](https://github.com/kowainik/relude/issues/178):
-  Made `die` be polymorphic in its return type.
+* [#171](https://github.com/kowainik/relude/issues/171):
+  Add custom type errors to various functions and instances.
+  (by [@vrom911](https://github.com/vrom911), [@chshersh](https://github.com/chshersh))
 * [#164](https://github.com/kowainik/relude/issues/164):
   Reexport `ShortByteString`, `toShort`/`fromShort` functions.
-* [#162](https://github.com/kowainik/relude/pull/162):
-  Use `foldr` instead of explicit recursion and `toList`.
+  (by [@vrom911](https://github.com/vrom911))
 * [#182](https://github.com/kowainik/relude/issues/182):
   Support GHC-8.8.1.
+  (by [@chshersh](https://github.com/chshersh))
+* [#168](https://github.com/kowainik/relude/pull/168),
+  [#197](https://github.com/kowainik/relude/pull/197):
+  Improve documentation significantly (more and better examples, better wording).
+  (by [@chshersh](https://github.com/chshersh),
+  [@vrom911](https://github.com/vrom911),
+  [@Cmdv](https://github.com/Cmdv))
+* [#177](https://github.com/kowainik/relude/issues/177):
+  Improve usage of performance pragmas.
+  (by [@chshersh](https://github.com/chshersh))
+* [#167](https://github.com/kowainik/relude/issues/167):
+  **Breaking change** Rename functions:
+    + `prec` to `prev`
+    + `dupe` to `dup`
```

Comment:
Should we add the migration guide?

Reply:
I think in this case it's quite obvious. And since we deprecate, there's also a message that tells what to do.